packages/shylu/shylu_dd/frosch/src/CoarseSpaces/FROSch_LocalPartitionOfUnityBasis_def.hpp

```diff
@@ -142,8 +142,8 @@ namespace FROSch {
         // Kann man das schöner machen?
```

Comment:
Of course one could ;-) .  (You could just delete such comments; they don't actually help readers.)

Reply:
You are right. This is just a note for myself ;)

Reply:
Ah, but once you put code in Trilinos, the code is no longer just for yourself, it's for the whole world :-)